homeassistant/components/fan/__init__.py

```diff
@@ -43,12 +43,12 @@
 
 ATTR_SPEED = 'speed'
 ATTR_SPEED_LIST = 'speed_list'
-ATTR_OSCILLATE = 'oscillate'
+ATTR_OSCILLATE = 'oscillating'
 
 PROP_TO_ATTR = {
     'speed': ATTR_SPEED,
     'speed_list': ATTR_SPEED_LIST,
-    'oscillate': ATTR_OSCILLATE,
+    'is_oscillating': ATTR_OSCILLATE,
```

Comment:
Wait - the property is `is_oscillating` but the state attribute will be `oscillating`